mustards_game/game.py

```diff
@@ -5,30 +5,24 @@
 from pygame.locals import MOUSEBUTTONDOWN
 from pygame.locals import QUIT
 
+import mustards_game.config as config
```

Comment:
why not 
```suggestion
from mustards_game.config import SCREEN_WIDTH, SCREEN_HEIGHT, MAX_ALTITUDE
```


then all following code changes are not necessary

Reply:
> why not
> 
> then all following code changes are not necessary

@beliaev-maksim import method changed